plasmapy/physics/parameters.py

```diff
@@ -296,7 +285,7 @@ def ion_sound_speed(*ignore,
 
     This function assumes that the product of the wavenumber and the
```

Comment:
Opened #316 to deal with generalizing this.

Reply:
Thank you!